### DISCUSSION TITLE
Server migration

### DISCUSSION THREAD

**Discussion by SerhiiZahuba** [bb-Ricardo/netbox-sync#26] [General]
1) There is a virtual server with a designated ip
2) I make it shutdown but do not delete
3) I am making a clone of a virtual machine to another server.
4) Start synchronization
5) The ip has not changed (it's just a server move)

When should an IP be assigned to a new virtual machine?

**Comment by bb-Ricardo** [comment]
Hi, this is what I've written in the README

>If the same IP address is found on a different interface (of a different device/VM) within the same realm (both using same VRF or both are global) then following test are performed:

>If the current interface is enabled and the new one disabled:
>  * IP stays at the current interface

>If the current interface ia disabled and the new one enabled:
>* IP will be moved to the enabled interface

>Both interfaces are in the same state (disabled/enable)
>* IP will also stay at the current interface as it's unclear which one would be the correct one
